libs/blocks/vimeo/vimeo.js

```diff
@@ -1,25 +1,80 @@
-import { createIntersectionObserver, createTag, isInTextNode } from '../../utils/utils.js';
+// part of the code is an optimized version of lite-vimeo-embed -> https://github.com/luwes/lite-vimeo-embed
+import { replaceKey } from '../../features/placeholders.js';
+import { createIntersectionObserver, createTag, getConfig, isInTextNode, loadLink } from '../../utils/utils.js';
 
-export default function init(a) {
-  if (isInTextNode(a)) return;
-  const embedVimeo = () => {
-    const url = new URL(a.href);
-    let src = url.href;
-    if (url.hostname !== 'player.vimeo.com') {
-      const video = url.pathname.split('/')[1];
-      src = `https://player.vimeo.com/video/${video}?app_id=122963`;
-    }
-    const iframe = createTag('iframe', {
-      src,
-      style: 'border: 0; top: 0; left: 0; width: 100%; height: 100%; position: absolute;',
+class LiteVimeo extends HTMLElement {
+  static preconnected = false;
+
+  connectedCallback() {
+    this.videoId = this.getAttribute('videoid');
+    this.language = this.getAttribute('language') || '';
```

Comment:
Is this property used anywhere, I cannot find it ?!

Reply:
No, it got left behind. I removed it 👍